.github/workflows/build-wheels.yml

```diff
@@ -310,6 +310,21 @@ jobs:
         run: |
           cargo clippy --workspace --all-features --all-targets
 
+  actionlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: actionlint
+        run: |
+          bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash)
```

Comment:
this looks a bit weird. it looks like we're surrendering our secret tokens to `rhysd`